airbyte-integrations/connectors/source-rki-covid/source_rki_covid/spec.json

```diff
@@ -0,0 +1,36 @@
+{
+  "documentationUrl": "https://docsurl.com",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Rki Covid Spec",
+    "type": "object",
+    "required": [],
+    "additionalProperties": false,
+    "properties": {
+      "cases_in_days": {
+        "type": "integer",
+        "description": "Number of cases days in the past from today."
+      },
+      "incidence_in_days": {
+        "type": "integer",
+        "description": "Number of incidence days in the past from today."
+      },
+      "deaths_in_days": {
+        "type": "integer",
+        "description": "Number of deaths days in the past from today."
+      },
+      "recovered_in_days": {
+        "type": "integer",
+        "description": "Number of recovered days in the past from today."
+      },
+      "frozen_incidence_in_days": {
+        "type": "integer",
+        "description": "Number of frozen-incidence days in the past from today."
+      },
+      "hospitalization_in_days": {
+        "type": "integer",
+        "description": "Number of hospitalization days in the past from today."
```

Comment:
Change those parameters to `start_date` and let users select the stream and receive data based on that.

Reply:
changed the parameters to start_date so that user can select the stream and receive data based on that.